src/cpu/flexcpu/simple_dataflow_thread.hh

```diff
@@ -50,19 +53,39 @@ class SDCPUThread
     class SDCPUExecContext : public ExecContext
     {
       protected:
-        SDCPUThread& backingState;
+        InflightInst& backingState;
```

Comment:
May want to rename this variable something like `dynamicInst` or just `inst` and add a comment.